src/wpt/harness.ts

```diff
@@ -66,8 +68,208 @@ type TestCase = {
   test(_: unknown, env: Env): Promise<void>;
 };
 
+type UnknownFunc = (...args: unknown[]) => unknown;
+
+enum TestPhase {
```

Comment:
enum is not recommended to be used since it has no direct translation to javascript. Can you convert this to an object?